docs/TestingAsyncCode.md

```diff
@@ -48,15 +48,13 @@ If `done()` is never called, the test will fail, which is what you want to happe
 
 ### Promises
```

Comment:
This section should really mention expect.assertions. cc @hramos, mind adding a line?